src/utils/index.ts

```diff
@@ -1,2 +1,3 @@
+export * from "./address"
```

Comment:
Shouldn't we export the `time` utils as well?

Reply:
50b5739a007c4883fb2056ba5c029fed48b7058f